gtk/src/toga_gtk/window.py

```diff
@@ -123,18 +97,28 @@ def close(self):
         self.native.close()
 
     def get_position(self):
-        pos = self.native.get_position()
-        return pos.root_x, pos.root_y
+        # Gtk believes/claims that positioning top level windows is not
+        # the toolkit’s job but WM job. They are suggesting leaving
+        # positioning to windowing system.
+        # See https://discourse.gnome.org/t/how-to-center-gtkwindows-in-gtk4/3112/4
```

Comment:
The API is not exist, so where we can document this discrepancy?

Reply:
It would be [here](https://toga.readthedocs.io/en/stable/reference/api/window.html); the Window API hasn't gone through a full audit yet, so it's documentation isn't completely up to date - but it would look something like [this section of the Button docs](https://toga.readthedocs.io/en/latest/reference/api/widgets/button.html#notes).